geode-docs/developing/expiration/configuring_data_expiration.html.md.erb

```diff
@@ -29,18 +29,22 @@ Configure the type of expiration and the expiration action to use.
 
 -   Set the expiration attributes by expiration type, with the max times and expiration actions. See the region attributes listings for `entry-time-to-live`, `entry-idle-time`, `region-time-to-live`, and `region-idle-time` in [&lt;region-attributes&gt;](../../reference/topics/cache_xml.html#region-attributes).
 
-    For partitioned regions, to ensure reliable read behavior, use the `time-to-live` attributes, not the `idle-time` attributes. In addition, you cannot use `local-destroy` or `local-invalidate` expiration actions in partitioned regions.
-
-    Replicated regions example:
-
-    ``` pre
-    // Setting standard expiration on an entry
-    <region-attributes statistics-enabled="true"> 
-      <entry-idle-time> 
-        <expiration-attributes timeout="60" action="local-invalidate"/> 
-      </entry-idle-time> 
-    </region-attributes> 
-    ```
+**For partitioned regions:**
```

Comment:
While I think that "For partitioned regions" helps to clarify the rest of this section, I'd still specify "partitioned region" wherever applicable for additional clarity, just in case someone is skimming or somehow reads a point out of context.